src/helpers/programHelper.js

```diff
@@ -800,6 +802,78 @@ class ProgramServiceHelper {
 
     return from(axios(req));
   }
+
+  /**
+   * Update the user profile with medium, subject and gradeLevel
+   *
+   * @param integer program_id  Program id
+   * @param integer user_id     User id
+   */
+  onAfterAddNomination(program_id, user_id) {
+    this.getProgramDetails(program_id).then(data => {
```

Comment:
Start using `async await` for promises. It is aesthetically better and we wont end up in this nested function. Take a look at `bulkJobService` for reference.

Reply:
As discussed closing this